Documentation/contributor-guide/release.md

```diff
@@ -66,28 +66,31 @@ which don't need to be executed before releasing each version.
 ### Release steps
 
 1. Raise an issue to publish the release plan, e.g. [issues/17350](https://github.com/etcd-io/etcd/issues/17350).
-2. Verify you can pass the authentication to the image registries,
+2. Raise a `kubernetes/org` pull request to temporarily elevate permissions for the GitHub release team.
+3. Once permissions are elevated, temporarily relax [branch protections](https://github.com/etcd-io/etcd/settings/branches) to allow pushing changes directly to `release-*` branches in GitHub.
```

Comment:
Is this also expected to be done by maintainers :( ?

Reply:
Neither step 2 or 3 have to be done by maintainers if new release team is established.  During release window the release team would have the temporary permissions to handle this. Refer: https://github.com/kubernetes/org/pull/5068

With that said I agree it's not ideal and updating release scripts to open a pr would be a nice idea if someone has some time to work on that.

Reply:
> Refer: [kubernetes/org#5068](https://github.com/kubernetes/org/pull/5068)

Please feel free to unhold the PR to let it be merged.



> open a pr would be a nice idea

Yes, it's good. But we do need directly pushing to upstream to push tags.